moztrap/model/mtapi.py

```diff
@@ -195,7 +195,7 @@ def delete_detail(self, request, **kwargs):
         return content.
         """
         res = super(MTResource, self).delete_detail(request, **kwargs)
-        del(res._headers["content-type"])
+        # del(res._headers["content-type"]) # Will cause DELETE fail
```

Comment:
If I uncomment this line and run the tests, I don't see an error.  Could you make a test to reproduce the delete error you're hitting?  I apologize, I know that is probably a pain.  But it would be really useful to cover it in the unit tests.  That way we can be sure we don't re-initiate it if we make further changes.

I also google'd around for the error this line is attempting to solve and ran into this.  It's not TastyPie (it's DjangoRestFramework instead) but similar issue.  I'm wondering if there's a condition where we COULD remove it without breaking deletes.

My motivation for this is that if we get too many warnings in the tests, it becomes harder to determine when something is going wrong vs. something that can be ignored.